registry/storage/driver/s3-aws/s3_test.go

```diff
@@ -767,6 +767,10 @@ func TestMoveWithMultipartCopy(t *testing.T) {
 }
 
 func TestListObjectsV2(t *testing.T) {
+	if skipS3() != "" {
```

Comment:
is this needed for gcs?

Reply:
Without this, if the S3 test env vars are not set this test is run regardless and fails. That shouldn't happen IMO.

Reply:
It's easy to trigger by settings the GCS env vars and running `BUILDTAGS=include_gcs make test`

Reply:
no issues, just wondering if it needs to go with gcs, but it's just a test check so it's fine. PR lgtm, i'll approve..

Reply:
This is actually now in three different PRs, doesn't matter, just means the first one that merges wins :)

Reply:
Let's take this out of this PR.